README.md

```diff
@@ -1,34 +1,23 @@
 # linter-htmlhint
 
-This linter plugin for [Linter](https://github.com/AtomLinter/Linter) provides
-an interface to [htmlhint](https://github.com/yaniswang/HTMLHint). It will be
+A plugin for [Linter](https://github.com/atom-community/linter) providing an interface to [HTMLHint]. It will be
```

Comment:
This should _just_ be `[Linter]` to use your single defined URL down below :wink:.